test/Microsoft.TestPlatform.AcceptanceTests/DisableAppdomainTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.AcceptanceTests
+{
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+
+    [TestClass]
+    public class DisableAppdomainTests : AcceptanceTestBase
```

Comment:
We already have test for this vstest/test/Microsoft.TestPlatform.AcceptanceTests/AppDomainTests.cs

Reply:
synced up offline. New test is doing more stuff. so retaining it